dynamic-layers/qt5-layer/recipes-qt/qt5/qtbase_%.bbappend

```diff
@@ -16,16 +16,33 @@ SRC_URI_append_imxgpu3d = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'x11', '', '${SRC_URI_APPEND_3D_NOT_X11}', d)} \
 "
 
+PACKAGECONFIG_GBM_IMXGPU_MX8_mx8 = "gbm kms"
+PACKAGECONFIG_GBM_IMXGPU_imxgpu  = "${PACKAGECONFIG_GBM_IMXGPU_MX8}"
+
 PACKAGECONFIG_GL_imxpxp   = "gles2"
-PACKAGECONFIG_GL_imxgpu2d = "${@bb.utils.contains('DISTRO_FEATURES', 'x11', ' gl', '', d)}"
-PACKAGECONFIG_GL_imxgpu3d = "gles2"
+PACKAGECONFIG_GL_imxgpu2d = "${@bb.utils.contains('DISTRO_FEATURES', 'x11', ' gl', '', d)} \
+                             ${PACKAGECONFIG_GBM_IMXGPU}"
+PACKAGECONFIG_GL_imxgpu3d = "gles2 \
+                             ${PACKAGECONFIG_GBM_IMXGPU}"
 PACKAGECONFIG_GL_use-mainline-bsp ?= "gles2 gbm kms"
 
+PACKAGECONFIG_PLATFORM_IMXGPU_MX8_mx8 = "eglfs"
+
 PACKAGECONFIG_PLATFORM          = ""
-PACKAGECONFIG_PLATFORM_imxgpu2d = "no-opengl linuxfb"
+PACKAGECONFIG_PLATFORM_imxgpu2d = " \
+    no-opengl linuxfb \
+    ${PACKAGECONFIG_PLATFORM_IMXGPU_MX8}"
```

Comment:
Why?

Reply:
Only for completeness, if ever we do have a 2D i.MX 8 part this will work and is likely to be correct. I can drop it if you'd prefer.

Reply:
Please drop. eglfs will fail to compile in this case, I believe.